src/sqlfluff/core/default_config.cfg

```diff
@@ -127,6 +127,9 @@ ignore_words_regex = None
 # Disabled for some dialects (e.g. bigquery)
 force_enable = False
 
+[sqlfluff:rules:L036]
+wildcard_policy = single
```

Comment:
To minimize disruption, I made `single` the default, since that was the old behavior (in spite of the docs saying otherwise).

Reply:
Good idea!